src/images/patterns/bg-000.svg

```diff
@@ -0,0 +1 @@
+<svg width="1" height="1" viewBox="0 0 1 1" xmlns="http://www.w3.org/2000/svg"><title>bg-000</title></svg>
```

Comment:
One last comment, because I am a garbage human being...
I think we'll also need to address this, since we're in the spirit of file renaming:
https://github.com/mozilla/Themer/blob/cf279fac5a7f1a81a937884a8569c61215585c42/src/extension/background.js#L57

Not sure if @lmorchard or anybody else knows of any other hard-coded file references or defaults. This was the only obvious thing that I could find doing some <kbd>$ git grep "bg-" | grep ".js:"</kbd>.